build.gradle.kts

```diff
@@ -67,7 +67,7 @@ fun getTag(): String {
         return systemVersion
     }
 
-    val gitVersion = shellExec("git describe --tags --dirty --long").trim()
+    val gitVersion = shellExec("/usr/bin/env git describe --tags --dirty --long").trim()
```

Comment:
What's the point of this ?
I did a test release https://github.com/hannesa2/js2d-gradle/releases/tag/3.0.1.Test without this and it's working well.
Local is working too without it

Reply:
aha, thanks. I thought it was calling a command without processing PATH for first argument (as I have to split it)